src/lib.rs

```diff
@@ -25,6 +25,7 @@ static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
 mod native;
 pub mod types;
+mod utils;
```

Comment:
Would you rather keep the existing `unsafe_alloc_buffer` defined in `src/lib.rs`? seems redundant

Reply:
Forgot to delete. Reason for this change is #55, which also needs it.

Reply:
Fixed.